2024-08-06 17:14:53,564 INFO    StreamThr :2082492 [internal.py:wandb_internal():86] W&B internal server running at pid: 2082492, started at: 2024-08-06 17:14:53.563918
2024-08-06 17:14:53,568 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: status
2024-08-06 17:14:53,570 INFO    WriterThread:2082492 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/run-tlmphxiz.wandb
2024-08-06 17:14:53,571 DEBUG   SenderThread:2082492 [sender.py:send():382] send: header
2024-08-06 17:14:53,571 DEBUG   SenderThread:2082492 [sender.py:send():382] send: run
2024-08-06 17:14:53,807 INFO    SenderThread:2082492 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files
2024-08-06 17:14:53,807 INFO    SenderThread:2082492 [sender.py:_start_run_threads():1136] run started: tlmphxiz with start time 1722935693.563892
2024-08-06 17:14:53,813 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: check_version
2024-08-06 17:14:53,813 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: check_version
2024-08-06 17:14:53,901 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: run_start
2024-08-06 17:14:53,906 DEBUG   HandlerThread:2082492 [system_info.py:__init__():32] System info init
2024-08-06 17:14:53,906 DEBUG   HandlerThread:2082492 [system_info.py:__init__():47] System info init done
2024-08-06 17:14:53,906 INFO    HandlerThread:2082492 [system_monitor.py:start():194] Starting system monitor
2024-08-06 17:14:53,906 INFO    SystemMonitor:2082492 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-06 17:14:53,906 INFO    HandlerThread:2082492 [system_monitor.py:probe():214] Collecting system info
2024-08-06 17:14:53,906 INFO    SystemMonitor:2082492 [interfaces.py:start():190] Started cpu monitoring
2024-08-06 17:14:53,907 INFO    SystemMonitor:2082492 [interfaces.py:start():190] Started disk monitoring
2024-08-06 17:14:53,907 INFO    SystemMonitor:2082492 [interfaces.py:start():190] Started gpu monitoring
2024-08-06 17:14:53,908 INFO    SystemMonitor:2082492 [interfaces.py:start():190] Started memory monitoring
2024-08-06 17:14:53,909 INFO    SystemMonitor:2082492 [interfaces.py:start():190] Started network monitoring
2024-08-06 17:14:53,916 DEBUG   HandlerThread:2082492 [system_info.py:probe():196] Probing system
2024-08-06 17:14:53,917 DEBUG   HandlerThread:2082492 [system_info.py:_probe_git():181] Probing git
2024-08-06 17:14:53,919 DEBUG   HandlerThread:2082492 [system_info.py:_probe_git():189] Probing git done
2024-08-06 17:14:53,919 DEBUG   HandlerThread:2082492 [system_info.py:probe():244] Probing system done
2024-08-06 17:14:53,919 DEBUG   HandlerThread:2082492 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-41-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-06T09:14:53.916617', 'startedAt': '2024-08-06T09:14:53.557116', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/Gemma-2B', '--use_lora', 'True', '--init_lora_weights', 'pissa_niter_4', '--target_modules', 'q_proj,o_proj,gate_proj,down_proj,up_proj', '--lora_rank', '32', '--lora_alpha', '32', '--lora_dropout', '0', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:395000]', '--output_dir', '/home/rwkv/JL/out_model/Gemma-meta-2btest', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.03', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': None}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1965.3065000000001, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 2248.097, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2117.842, 'min': 800.0, 'max': 5500.0}, {'current': 2224.269, 'min': 800.0, 'max': 5500.0}, {'current': 2817.137, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 2863.508, 'min': 800.0, 'max': 5500.0}, {'current': 2242.498, 'min': 800.0, 'max': 5600.0}, {'current': 1821.946, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 2441.313, 'min': 800.0, 'max': 5600.0}, {'current': 2241.914, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2467.825, 'min': 800.0, 'max': 5500.0}, {'current': 5500.014, 'min': 800.0, 'max': 5500.0}, {'current': 3000.714, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 440.378719329834}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.5532341003418}}
2024-08-06 17:14:53,919 INFO    HandlerThread:2082492 [system_monitor.py:probe():224] Finished collecting system info
2024-08-06 17:14:53,919 INFO    HandlerThread:2082492 [system_monitor.py:probe():227] Publishing system info
2024-08-06 17:14:53,919 DEBUG   HandlerThread:2082492 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-06 17:14:53,920 DEBUG   HandlerThread:2082492 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-06 17:14:53,920 DEBUG   HandlerThread:2082492 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-06 17:14:54,807 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/conda-environment.yaml
2024-08-06 17:14:54,807 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/requirements.txt
2024-08-06 17:14:55,598 DEBUG   HandlerThread:2082492 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-06 17:14:55,598 DEBUG   HandlerThread:2082492 [system_info.py:_save_code():90] Saving code
2024-08-06 17:14:55,600 DEBUG   HandlerThread:2082492 [system_info.py:_save_code():111] Saving code done
2024-08-06 17:14:55,600 DEBUG   HandlerThread:2082492 [system_info.py:_save_patches():128] Saving git patches
2024-08-06 17:14:55,602 ERROR   HandlerThread:2082492 [system_info.py:_save_patches():169] Error generating diff: Reference at 'refs/remotes/origin/main' does not exist
2024-08-06 17:14:55,602 DEBUG   HandlerThread:2082492 [system_info.py:_save_patches():170] Saving git patches done
2024-08-06 17:14:55,603 INFO    HandlerThread:2082492 [system_monitor.py:probe():229] Finished publishing system info
2024-08-06 17:14:55,603 DEBUG   SenderThread:2082492 [sender.py:send():382] send: files
2024-08-06 17:14:55,603 INFO    SenderThread:2082492 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-06 17:14:55,604 INFO    SenderThread:2082492 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-06 17:14:55,606 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 17:14:55,606 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: stop_status
2024-08-06 17:14:55,608 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 17:14:55,792 DEBUG   SenderThread:2082492 [sender.py:send():382] send: telemetry
2024-08-06 17:14:55,792 DEBUG   SenderThread:2082492 [sender.py:send():382] send: config
2024-08-06 17:14:55,792 DEBUG   SenderThread:2082492 [sender.py:send():382] send: metric
2024-08-06 17:14:55,792 DEBUG   SenderThread:2082492 [sender.py:send():382] send: telemetry
2024-08-06 17:14:55,792 DEBUG   SenderThread:2082492 [sender.py:send():382] send: metric
2024-08-06 17:14:55,792 WARNING SenderThread:2082492 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-06 17:14:55,792 DEBUG   SenderThread:2082492 [sender.py:send():382] send: telemetry
2024-08-06 17:14:55,792 DEBUG   SenderThread:2082492 [sender.py:send():382] send: telemetry
2024-08-06 17:14:55,792 DEBUG   SenderThread:2082492 [sender.py:send():382] send: config
2024-08-06 17:14:55,807 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/conda-environment.yaml
2024-08-06 17:14:55,807 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/code/llama/gmm.py
2024-08-06 17:14:55,807 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/wandb-metadata.json
2024-08-06 17:14:55,807 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/output.log
2024-08-06 17:14:55,807 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/code
2024-08-06 17:14:55,807 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/code/llama
2024-08-06 17:14:55,880 DEBUG   SenderThread:2082492 [sender.py:send():382] send: exit
2024-08-06 17:14:55,880 INFO    SenderThread:2082492 [sender.py:send_exit():589] handling exit code: 1
2024-08-06 17:14:55,880 INFO    SenderThread:2082492 [sender.py:send_exit():591] handling runtime: 1
2024-08-06 17:14:55,882 INFO    SenderThread:2082492 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 17:14:55,883 INFO    SenderThread:2082492 [sender.py:send_exit():597] send defer
2024-08-06 17:14:55,883 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:55,883 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-06 17:14:55,884 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:55,884 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-06 17:14:55,884 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 1
2024-08-06 17:14:55,884 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:55,884 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-06 17:14:55,885 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:55,885 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-06 17:14:55,885 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 2
2024-08-06 17:14:55,885 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:55,885 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-06 17:14:55,885 INFO    HandlerThread:2082492 [system_monitor.py:finish():203] Stopping system monitor
2024-08-06 17:14:55,885 DEBUG   SystemMonitor:2082492 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-06 17:14:55,885 DEBUG   SystemMonitor:2082492 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-06 17:14:55,885 DEBUG   SystemMonitor:2082492 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-06 17:14:55,886 INFO    HandlerThread:2082492 [interfaces.py:finish():202] Joined cpu monitor
2024-08-06 17:14:55,887 INFO    HandlerThread:2082492 [interfaces.py:finish():202] Joined disk monitor
2024-08-06 17:14:55,888 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1
2024-08-06 17:14:55,888 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5476', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:55,889 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E918E30A928005</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:55,942 INFO    HandlerThread:2082492 [interfaces.py:finish():202] Joined gpu monitor
2024-08-06 17:14:55,942 INFO    HandlerThread:2082492 [interfaces.py:finish():202] Joined memory monitor
2024-08-06 17:14:55,942 INFO    HandlerThread:2082492 [interfaces.py:finish():202] Joined network monitor
2024-08-06 17:14:55,943 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 3
2024-08-06 17:14:55,943 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:55,943 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-06 17:14:55,943 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 4
2024-08-06 17:14:55,943 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:55,943 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-06 17:14:55,943 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 5
2024-08-06 17:14:55,943 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:55,943 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-06 17:14:55,943 DEBUG   SenderThread:2082492 [sender.py:send():382] send: summary
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 17:14:55,943 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-06 17:14:55,943 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 6
2024-08-06 17:14:55,943 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:55,943 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-06 17:14:55,944 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:55,944 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-06 17:14:55,945 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 17:14:56,031 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d
2024-08-06 17:14:56,031 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:56,031 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E918E30F0ECC3A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:56,052 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 7
2024-08-06 17:14:56,053 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:56,053 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-06 17:14:56,053 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:56,053 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-06 17:14:56,808 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/config.yaml
2024-08-06 17:14:56,808 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/wandb-summary.json
2024-08-06 17:14:56,873 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:14:56,989 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1
2024-08-06 17:14:56,989 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5476', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:56,989 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E918E34BCE1F94</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:57,357 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d
2024-08-06 17:14:57,358 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:57,358 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E918E360E3981F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:57,358 INFO    wandb-upload_1:2082492 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 17:14:57,806 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 8
2024-08-06 17:14:57,806 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:14:57,806 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:57,806 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-06 17:14:57,806 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:57,806 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-06 17:14:57,806 INFO    SenderThread:2082492 [job_builder.py:build():298] Attempting to build job artifact
2024-08-06 17:14:57,807 INFO    SenderThread:2082492 [job_builder.py:_get_source_type():439] no source found
2024-08-06 17:14:57,807 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 9
2024-08-06 17:14:57,807 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:57,807 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-06 17:14:57,807 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:57,807 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 9
2024-08-06 17:14:57,807 INFO    SenderThread:2082492 [dir_watcher.py:finish():358] shutting down directory watcher
2024-08-06 17:14:57,808 INFO    Thread-12 :2082492 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/output.log
2024-08-06 17:14:57,808 INFO    SenderThread:2082492 [dir_watcher.py:finish():388] scan: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files
2024-08-06 17:14:57,808 INFO    SenderThread:2082492 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/wandb-summary.json wandb-summary.json
2024-08-06 17:14:57,808 INFO    SenderThread:2082492 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/requirements.txt requirements.txt
2024-08-06 17:14:57,808 INFO    SenderThread:2082492 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/output.log output.log
2024-08-06 17:14:57,808 INFO    SenderThread:2082492 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/config.yaml config.yaml
2024-08-06 17:14:57,808 INFO    SenderThread:2082492 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/wandb-metadata.json wandb-metadata.json
2024-08-06 17:14:57,809 INFO    SenderThread:2082492 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/conda-environment.yaml conda-environment.yaml
2024-08-06 17:14:57,809 INFO    SenderThread:2082492 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/code/llama/gmm.py code/llama/gmm.py
2024-08-06 17:14:57,809 INFO    SenderThread:2082492 [sender.py:transition_state():617] send defer: 10
2024-08-06 17:14:57,809 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: defer
2024-08-06 17:14:57,809 INFO    HandlerThread:2082492 [handler.py:handle_request_defer():172] handle defer: 10
2024-08-06 17:14:57,811 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: defer
2024-08-06 17:14:57,812 INFO    SenderThread:2082492 [sender.py:send_request_defer():613] handle sender defer: 10
2024-08-06 17:14:57,812 INFO    SenderThread:2082492 [file_pusher.py:finish():175] shutting down file pusher
2024-08-06 17:14:57,874 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:14:57,875 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:14:57,943 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd
2024-08-06 17:14:57,943 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:57,943 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E918E38494A6CF</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:57,969 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=eca4ec342055f431b9c6b5b13d8e48bb16b52e6a6eae9d93443b8753a2ebb147: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=eca4ec342055f431b9c6b5b13d8e48bb16b52e6a6eae9d93443b8753a2ebb147
2024-08-06 17:14:57,970 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10561', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:57,970 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/output.log</Resource><Region>wandb-local</Region><RequestId>17E918E3861BC9A4</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:57,983 INFO    wandb-upload_2:2082492 [upload_job.py:push():131] Uploaded file /home/rwkv/JL/GMM/llama/wandb/run-20240806_171453-tlmphxiz/files/wandb-summary.json
2024-08-06 17:14:58,051 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a569ae0008ef4c94a3735d693a12b00d0379cbcaf7e1a4bc8828a8c277f57644: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a569ae0008ef4c94a3735d693a12b00d0379cbcaf7e1a4bc8828a8c277f57644
2024-08-06 17:14:58,051 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '12014', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:58,051 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E918E38A275C10</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:58,055 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b728fc5f324d9b9665223678c0ce0f21618c48eb0c72184c8f3bdf5a3e1d43b1: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b728fc5f324d9b9665223678c0ce0f21618c48eb0c72184c8f3bdf5a3e1d43b1
2024-08-06 17:14:58,055 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:58,055 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E918E38B8C2482</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:58,875 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:14:58,876 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:14:59,042 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd
2024-08-06 17:14:59,042 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:59,042 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E918E3C60AE576</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:59,042 INFO    wandb-upload_3:2082492 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 17:14:59,082 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1
2024-08-06 17:14:59,082 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5476', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:59,082 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E918E3C8794FA4</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:59,082 INFO    wandb-upload_0:2082492 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 17:14:59,220 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=eca4ec342055f431b9c6b5b13d8e48bb16b52e6a6eae9d93443b8753a2ebb147: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=eca4ec342055f431b9c6b5b13d8e48bb16b52e6a6eae9d93443b8753a2ebb147
2024-08-06 17:14:59,220 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10561', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:59,220 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/output.log</Resource><Region>wandb-local</Region><RequestId>17E918E3D03A760B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:59,355 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b728fc5f324d9b9665223678c0ce0f21618c48eb0c72184c8f3bdf5a3e1d43b1: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b728fc5f324d9b9665223678c0ce0f21618c48eb0c72184c8f3bdf5a3e1d43b1
2024-08-06 17:14:59,355 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:59,355 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E918E3D83AF764</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:59,363 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a569ae0008ef4c94a3735d693a12b00d0379cbcaf7e1a4bc8828a8c277f57644: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a569ae0008ef4c94a3735d693a12b00d0379cbcaf7e1a4bc8828a8c277f57644
2024-08-06 17:14:59,363 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '12014', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:59,363 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E918E3D86E3025</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:59,678 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d
2024-08-06 17:14:59,678 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:14:59,678 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E918E3EB2EDBE6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:14:59,877 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:14:59,878 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:00,878 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:00,879 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:01,334 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd
2024-08-06 17:15:01,334 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:01,334 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E918E44EAB4023</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:01,537 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a569ae0008ef4c94a3735d693a12b00d0379cbcaf7e1a4bc8828a8c277f57644: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a569ae0008ef4c94a3735d693a12b00d0379cbcaf7e1a4bc8828a8c277f57644
2024-08-06 17:15:01,537 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '12014', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:01,537 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E918E459D33B6C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:01,560 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=eca4ec342055f431b9c6b5b13d8e48bb16b52e6a6eae9d93443b8753a2ebb147: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=eca4ec342055f431b9c6b5b13d8e48bb16b52e6a6eae9d93443b8753a2ebb147
2024-08-06 17:15:01,560 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10561', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:01,560 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/output.log</Resource><Region>wandb-local</Region><RequestId>17E918E45B5071B2</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:01,776 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b728fc5f324d9b9665223678c0ce0f21618c48eb0c72184c8f3bdf5a3e1d43b1: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b728fc5f324d9b9665223678c0ce0f21618c48eb0c72184c8f3bdf5a3e1d43b1
2024-08-06 17:15:01,777 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:01,777 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E918E4687C0C82</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:01,881 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:01,881 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:02,881 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:02,882 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:03,786 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=28da509a5227d5bd33e507965311040e34bcde542e5cb8f8058b6d57527e9a5d
2024-08-06 17:15:03,786 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:03,786 ERROR   wandb-upload_1:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E918E4DFF6399D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:03,883 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:03,884 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 17:15:03,884 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:03,927 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091455Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d3ab3d4f97d38f90999ebff812b40b7786a1e3811fbeb6546a70fce3fd3dccb1
2024-08-06 17:15:03,927 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5476', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:03,927 ERROR   wandb-upload_0:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E918E4E946D975</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:04,885 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:04,886 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:05,739 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a569ae0008ef4c94a3735d693a12b00d0379cbcaf7e1a4bc8828a8c277f57644: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a569ae0008ef4c94a3735d693a12b00d0379cbcaf7e1a4bc8828a8c277f57644
2024-08-06 17:15:05,739 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '12014', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:05,739 ERROR   wandb-upload_5:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E918E554668C01</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:05,752 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=6c232afe96a970a9ba975559f9263816cec5c35b6d106fd50fa3af129fa033dd
2024-08-06 17:15:05,752 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:05,752 ERROR   wandb-upload_3:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E918E556437008</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:05,778 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=eca4ec342055f431b9c6b5b13d8e48bb16b52e6a6eae9d93443b8753a2ebb147: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=eca4ec342055f431b9c6b5b13d8e48bb16b52e6a6eae9d93443b8753a2ebb147
2024-08-06 17:15:05,779 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10561', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:05,779 ERROR   wandb-upload_4:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/output.log</Resource><Region>wandb-local</Region><RequestId>17E918E556F3073F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:05,888 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:05,888 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:06,223 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b728fc5f324d9b9665223678c0ce0f21618c48eb0c72184c8f3bdf5a3e1d43b1: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T091457Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b728fc5f324d9b9665223678c0ce0f21618c48eb0c72184c8f3bdf5a3e1d43b1
2024-08-06 17:15:06,223 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 17:15:06,223 ERROR   wandb-upload_6:2082492 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/tlmphxiz/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/tlmphxiz/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E918E5718EF660</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 17:15:06,888 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:06,889 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:07,890 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:07,891 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:08,891 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 17:15:08,892 DEBUG   SenderThread:2082492 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 17:15:08,892 DEBUG   HandlerThread:2082492 [handler.py:handle_request():146] handle_request: status_report
